src/components/MusicTile/MusicTile.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import {
+  ImgWrapper,
+  Wrapper,
+  TitlesWrapper,
+  Title,
+  SubTitle,
+  RemoveButton,
+} from "./styles";
+import X_Icon from "../../assets/icons/x_icon.svg";
+
+export interface MusicTileProps {
+  img: string;
+  title: string;
+  subtitle: string;
+  imageAlt: string;
+  index: number;
+  handleRemoveButtonClick: Function;
+}
+
+export const MusicTile = (props: MusicTileProps) => {
+  const { img, title, subtitle, imageAlt, index, handleRemoveButtonClick } =
+    props;
+
+  const [showRemoveButton, setShowRemoveButton] = useState(false);
+  const handleMouseEnter = () => {
+    setShowRemoveButton(true);
+  };
+  const handleMouseLeave = () => {
+    setShowRemoveButton(false);
+  };
+
+  return (
+    <Wrapper onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
+      {showRemoveButton && (
+        <RemoveButton
+          onClick={() => {
+            handleRemoveButtonClick(index);
+          }}
+        >
+          <img src={X_Icon} draggable="false" />
```

Comment:
You’re using an SVG with an `img` tag, just curious, do you know of another way of rendering SVGs in react?